scalability_and_performance/ztp-deploying-disconnected.adoc

```diff
@@ -28,35 +28,49 @@ include::modules/ztp-du-host-networking-requirements.adoc[leveloffset=+1]
-
-include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
-
-.Additional information
-
-For information on viewing the CRI-O logs to view the image source, see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+Before you install a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You can also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
 
 [IMPORTANT]
 ====
 You must have access to the internet to obtain the necessary container images.
 In this procedure, you place the mirror registry on a mirror host
-that has access to both your network and the internet. If you do not have access
+that has access to both your network and the Internet. If you do not have access
 to a mirror host, use the disconnected procedure to copy images to a device you
 can move across network boundaries with.
 ====
 
-include::modules/installation-creating-mirror-registry.adoc[leveloffset=+2]
+[id="prerequisites_installing-mirroring-installation-images_{context}"]
+=== Prerequisites
+
+* You must have a container image registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2/[Docker v2-2] in the location that will host the {product-title} cluster, such as one of the following registries:
++
+--
+** link:https://www.redhat.com/en/technologies/cloud-computing/quay[Red Hat Quay]
+** link:https://jfrog.com/artifactory/[JFrog Artifactory]
+** link:https://www.sonatype.com/products/repository-oss?topnav=true[Sonatype Nexus Repository]
+** link:https://goharbor.io/[Harbor]
+--
+If you have an entitlement to Red Hat Quay, see the documentation on deploying Red Hat Quay link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_for_proof-of-concept_non-production_purposes/[for proof-of-concept purposes] or link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html/deploy_red_hat_quay_on_openshift_with_the_quay_operator/[by using the Quay Operator]. If you need additional assistance selecting and installing a registry, contact your sales representative or Red Hat support.
+
+include::modules/installation-about-mirror-registry.adoc[leveloffset=+2]
+
+.Additional information
+
+For information on viewing the CRI-O logs to view the image source, see xref:../installing/validating-an-installation.html#viewing-the-image-pull-source_validating-an-installation[Viewing the image pull source].
+
+[id="installing-preparing-mirror_{context}"]
+=== Preparing your mirror host
+
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
 
-include::modules/cli-installing-cli.adoc[leveloffset=+2]
+include::modules/cli-installing-cli.adoc[leveloffset=+3]
```

Comment:
This isn't in the scope of this PR, but I am a little curious as to why this procedure (or rather, set of procedures) is included here as a step within the "Preparing your mirror host" procedure. Could it be relocated as one of the prerequisites with a link to the `oc` install docs (https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html#installing-openshift-cli)?

Reply:
I'm just reusing the procedure as documented here: https://docs.openshift.com/container-platform/4.8/installing/installing-mirroring-installation-images.html

The user needs to install `oc` on the mirror host.